observability/logging/logging-6.0/log6x-loki.adoc

```diff
@@ -0,0 +1,105 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="log6x-loki"]
+= Storing logs with LokiStack
+include::_attributes/common-attributes.adoc[]
+:context: logging-6x
+
+toc::[]
+
+== Installation
+Partial includes of modules/logging-loki-cli-install.adoc & logging-loki-gui-install.adoc once https://github.com/openshift/openshift-docs/pull/77407/ is merged.
+
+
+
+
+// All of these need to be validated by engineering for 6.0.
+include::modules/logging-enabling-loki-alerts.adoc[leveloffset=+1]
+
+include::modules/logging-loki-memberlist-ip.adoc[leveloffset=+1]
+
+//include::modules/logging-loki-log-access.adoc[leveloffset=+1]
+
+include::modules/logging-identity-federation.adoc[leveloffset=+1]
+
+include::modules/logging-loki-log-access.adoc[leveloffset=+1]
+
+include::modules/logging-loki-pod-placement.adoc[leveloffset=+1]
+
+include::modules/logging-loki-reliability-hardening.adoc[leveloffset=+1]
+
+include::modules/logging-loki-restart-hardening.adoc[leveloffset=+1]
+
+include::modules/logging-loki-retention.adoc[leveloffset=+1]
+
+include::modules/logging-loki-zone-aware-rep.adoc[leveloffset=+1]
+
+include::modules/logging-loki-zone-fail-recovery.adoc[leveloffset=+1]
+
+include::modules/loki-rate-limit-errors.adoc[leveloffset=+1]
+
+include::modules/loki-rbac-rules-permissions.adoc[leveloffset=+1]
+
```

Comment:
Eng only needs to verify the include modules above this line. The content below this line is new.

Reply:
@libander, can I remove the new content `Loki Architecture: Understanding the Components`  from this PR to avoid any confusion. I will proceed with verification of the moduels only.

Reply:
@shipsing - Absolutely. I am 100% onboard with clarity.